### PR TITLE
Added support for UTF-8

### PR DESCRIPTION
**Pull request by deveshmittalpaytm**
Added support for Unicode characters such as '\u20b9'

### CODE REVIEW

**Comment by deveshmittalpaytm** [comment]
I signed it!

**Comment by deveshmittalpaytm** [comment]
Patch seemingly had my devesh.mittal@paytm.com account. 
Closing this PR and raising another.

File: config/configure.py [hunk 1]
```diff
@@ -50,14 +51,14 @@ def _publish(etag):
     etag: ETag for safe (avoid race conditions) template updates.
         * can be used to force template replacement.
   """
-  with open('config.json', 'r') as f:
+  with open('config.json', 'r', encoding="utf-8") as f:
```
Comment (kroikie) [inline config/configure.py]: Would you mind using single quotes here?

File: config/configure.py [hunk 2]
```diff
@@ -33,8 +34,8 @@ def _get():
   resp = requests.get(REMOTE_CONFIG_URL, headers=headers)
 
   if resp.status_code == 200:
-    with open('config.json', 'w') as f:
-      f.write(resp.text)
+    with io.open('config.json', 'wb') as f:
+      f.write(resp.text.encode('utf8'))
```
Comment (kroikie) [inline config/configure.py]: Please change to `utf-8` so it is consistent.

**Pull request merged**